src/device/usbd.c

```diff
@@ -764,10 +764,8 @@ static bool process_set_config(uint8_t rhport, uint8_t cfg_num)
         if (desc_itf_assoc)
         {
           // IAD's first interface number and class/subclass/protocol should match with opened interface
```

Comment:
a bit of annoying, but could you also update the command to only interface number and class should match ( drop subclas/protocol).